include stdlib.h instead of malloc.h, since the latter may not be
available, or be somewhere else, while the former should be always
around.  This is in particular a problem on Darwin and *BSD.
http://bugs.gentoo.org/show_bug.cgi?id=292187

--- boxbackup-0.10/lib/backupclient/BackupStoreFile.h
+++ boxbackup-0.10/lib/backupclient/BackupStoreFile.h
@@ -53,7 +53,7 @@
 #include "BackupStoreFilename.h"
 
 #include <memory>
-#include <malloc.h>
+#include <stdlib.h>
 
 typedef struct 
 {

--- boxbackup-0.10/lib/common/WaitForEvent.cpp
+++ boxbackup-0.10/lib/common/WaitForEvent.cpp
@@ -53,7 +53,7 @@
 
 #include <errno.h>
 #include <string.h>
-#include <malloc.h>
+#include <stdlib.h>
 
 #include "WaitForEvent.h"
 
--- boxbackup-0.10/lib/raidfile/RaidFileRead.cpp
+++ boxbackup-0.10/lib/raidfile/RaidFileRead.cpp
@@ -59,7 +59,7 @@
 
 #include <stdio.h>
 #include <string.h>
-#include <malloc.h>
+#include <stdlib.h>
 #include <memory>
 #include <map>
 
